manifester/manifester.py

```diff
@@ -67,6 +67,20 @@ def create_subscription_allocation(self):
         )
         return self.allocation_uuid
 
+    def delete_subscription_allocation(self):
+        self._access_token = None
+        data = {
+            "headers": {"Authorization": f"Bearer {self.access_token}"},
+            "proxies": self.manifest_data.get("proxies", settings.proxies),
+            "params": {"force": "true"},
+        }
+        response = simple_retry(
+            requests.delete,
+            cmd_args=[f"{self.manifest_data.url.allocations}/{self.allocation_uuid}"],
```

Comment:
So does your settings object have a dictionary called `url` that has the endpoints listed? If so should we consider adding placeholders into the example settings file for the url dict? Same here applies for the `proxies` on L74

Reply:
We absolutely should for the `url` dictionary, and in fact they are already present in the example settings: https://github.com/SatelliteQE/manifester/blob/e5e4a90d5facc529133960f7002615fcbb145610/manifester_settings.yaml.example#L20

I don't think we should add a placeholder for the `proxies` setting, though. My hunch is that most users will not be using a proxy with manifester, in which case the value of the `proxies` setting should be an empty string.. The `proxies` setting was added to enable support for the internal Red Hat stage environment because calls to the stage API must be made through an internal proxy. Since both the stage environment and the proxy are Red Hat-internal and this is a public project, we should not include those URLs in the example settings.

Reply:
Ohhhh it's under the `golden_ticket` entry. Reading non-highlighted text is difficult.

Reply:
Yeah, the `url` settings are per-manifest category, so a user could conceivably have multiple manifests defined that are getting their subscriptions from different environments.